.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - run:
           name: Update libraries
           command: |
-            sudo apt-get update
+            sudo apt-get --allow-releaseinfo-change update
```

Comment:
I fixed this elsewhere recently by changing to `sudo apt` I think, but this is also fine

Reply:
Updated to sudo apt.